axelrod/match_generator.py

```diff
@@ -191,11 +217,7 @@ class SpatialMatches(RoundRobinMatches):
 
     def __init__(self, players, turns, game, repetitions, edges):
 
-        player_indices = list(range(len(players)))
-        node_indices = sorted(set([node for edge in edges for node in edge]))
-        if player_indices != node_indices:
-            raise ValueError("The graph edges do not include all players.")
-
+        test_graph_is_connected(edges, players)
```

Comment:
I know that means a slight repeat of code, but it makes the function more readily understood, I reckon